app/routes.js

```diff
@@ -24,6 +24,16 @@ module.exports = function(app) {
             .exec(resHandler(res));
     });
 
+    // find unique values for given screen and field
+    app.get('/api/unique/:screen/:field', function(req, res) {
+        Sample
+            .find({'screen': req.params.screen})
+            .distinct(req.params.field, function(err, data) {
```

Comment:
Long answer:
JS actually doesn't check the number of arguments passed.. normally optional arguments are handled something like this.

```
function excellentFunction(optionalArgument) {
    if(optionalArgument === undefined) {
        optionalArgument = 'defaultValue';
    }
    return optionalArgument;
}
```

It's one of the annoyances of ES5 (the current version of JS), though optional arguments are coming in ES6 which is pretty exciting.

Short answer:
Yeah that can be done :)